### DISCUSSION TITLE
Label Text Right position

### DISCUSSION THREAD

**Discussion by MarceloRab** [darioielardi/flutter_speed_dial#151] [Q&A] [answered]
For a button on the left side of the screen, how do we move the SpeedDialChild texts to the right?

For now it's like this.

```dart
labelWidget: Transform.translate(
   offset: Offset(120, 0),
   child: Text(
     "test 1",
     style: ConsultaAppTheme.titleDefault,
   ),
),
```

**Comment by prateekmedia** [comment, marked as answer]
Checkout the new version, it has everything fixed. To use this you have to set the value of `switchLabelPosition` to true.
